content/guides/general/inline-blocks/inline-form-fields.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Add an Inline Form & Fields
+---
+
+## Step 2 - Register a generic form
+
+<!-- Add tagged commit -->
+
+The first thing we are going to do is create an _Inline Form_ with [`useForm`](<[https://tinacms.org/docs/forms#creating-forms-in-react](https://tinacms.org/docs/forms#creating-forms-in-react)>). Note that since we don't have a backend set up, the data changes won't persist and the toolbar won't save / track form status as expected. Please refer to [other guides](<[https://tinacms.org/guides/](https://tinacms.org/guides/)>) to learn more on setting up a backend.
```

Comment:
Is it just me or are all these links messed up?

Reply:
probably formatting issues from the export